.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=120
```

Comment:
Is it common to have this kind of configuration as part of the repository? I feel like this should be left to each of us to decide.

Reply:
You're right, I'll revert that.

Reply:
done.